crates/compilers/src/cache.rs

```diff
@@ -411,6 +418,8 @@ pub struct CacheEntry {
     pub last_modification_date: u64,
     /// hash to identify whether the content of the file changed
     pub content_hash: String,
+    /// hash of the interface representation of the file, if it's a source file
+    pub interface_repr_hash: Option<String>,
```

Comment:
nit: noticed `interfaceReprHash` is sometimes `null` in the cache, would it make sense to skip this field when it is `None` during serialization w/ `#[serde(skip_serializing_if = "Option::is_none")]`?

![Screenshot from 2025-03-05 12-45-41](https://github.com/user-attachments/assets/8ffbd54f-6d45-485d-bdcf-3b1927f39699)

Reply:
yep, that's null for all files under test / script dirs, will check if any side effect if we don't serialize them but shouldn't